README.md

```diff
@@ -1,3 +1,5 @@
+> **_NOTE:_** Add SECRET_KEY="sklajdgb34tbksdjavfSECREt" to env in development for jwt functions
```

Comment:
It doesn't matter what secret content is it right?

A good practice is to create a `.env.default` file that contains all the ENV vars filled in, for the ones that are meaningful you put a template, for the ones that aren't like this one you add the key already.

Reply:
and that default i can add to the commit so everybody pulling will have the dev default config right?

Reply:
Yes, the idea is to share the template. Any real secrets should not be commited.